### ISSUE TITLE
Breaking out team update, feedstocks update, and feedstock listing update 

### ISSUE THREAD

**Issue by jakirkham** [conda-forge/conda-forge.github.io#214]
It is pretty silly we trigger a CI job to regenerate update teams, the feedstock repo, and the feedstock listing every time we merge here given what we are merging (pinnings, minutes, general docs, etc.) which have nothing to do with the feedstocks. We also then have to watch senseless build failures show up and get notifications about them; when, they are unrelated to the bulk of the content here. Not to mention we end up having to clone the feedstock repo in some cases to get the job done.

What we need is a better separation of concerns. The feedstocks repo should have CI and all of the feedstock info like this feedstock listing. As the feedstocks repo does not have the same activity as this one and that activity is strictly related to the feedstocks repo itself, this should cutdown on extra runs of these feedstock update actions when unintended. For now the relevant scripts can move over there, but we may want to break them out in the long run.

Initially it may seem concerning that part of the webpage would be moving (the feedstock listing). However, as we have seen by linking the status page in PR ( https://github.com/conda-forge/conda-forge.github.io/pull/210 ), linking of different repos webpages is seamless. We could simply turn `feedstocks.html` into `index.html` and thus ( http://conda-forge.github.io/feedstocks.html ) would become ( http://conda-forge.github.io/feedstocks ). In fact that latter link (sans `.html`) works now. Thus this would actually work quite nicely.

In addition to these benefits, we would cutdown or CI usage because of lower activity. We would also no longer need a matrix for doing the update as all contents will live in the same repo. We would actually be able to use CI at this repo for anything we want as we will have freed up that resource. Our feedstock listing and feedstock submodules would remain in sync (unlike now). The overall complexity of our setup would be reduced and thus our management overhead will be reduced.

To summarize, we should break out feedstock maintenance stuff and move it to the feedstocks repo to simplify our workflow.

**Comment by jakirkham** [comment]
Added PR ( https://github.com/conda-forge/conda-forge.github.io/pull/216 ) to switch from `feedstocks.html` to `feedstocks` in links. This should work basically the same. It just prepares us for the move. Though it doesn't commit us to it.

**Comment by jakirkham** [comment]
What do you think about this, @pelson?

**Comment by jakirkham** [comment]
Have enabled Travis CI on the feedstocks repo. Also have ensured both the feedstocks repo and this repo (the webpage repo) have `Build only if .travis.yml is present` enabled. Thus the feedstocks repo is not running CI yet and the webpage repo won't run CI if we remove the `.travis.yml` file.

**Comment by jakirkham** [comment]
See PR ( https://github.com/conda-forge/feedstocks/pull/6 ) for the start of this work.

**Comment by jakirkham** [comment]
I'm starting to rethink this. Basically our global update strategy feels very inefficient and will eventually breakdown at larger scales if for no other reason than the GitHub API rate limit. A better strategy would just to make a Heroku job that listens for push events on repos and checks for maintainer changes. It can also update the feedstocks repo at the same time. This will keep things in better sync for less.

**Comment by jakirkham** [comment]
Please see issue ( https://github.com/conda-forge/conda-forge-webservices/issues/63 ) for a write-up on how this service might work.

Note: It may still be worthwhile to move the feedstock listing page anyways.

**Comment by jakirkham** [comment]
Team additions are now handled with a webservice thanks to @isuruf's work in PR ( https://github.com/conda-forge/conda-forge-webservices/pull/96 ) and other subsequent PRs. Also we no longer trigger team updates at this repo now that PR ( https://github.com/conda-forge/conda-forge.github.io/pull/453 ) is accepted.

Updating the feedstocks repo is now handled by a webservice thanks to PR ( https://github.com/conda-forge/conda-forge-webservices/pull/115 ). With PR ( https://github.com/conda-forge/conda-forge.github.io/pull/458 ), we will no longer handle feedstock updates on this repo.

The only remaining question is how to handle the feedstock listing on the webpage going forward. Have opened issue ( https://github.com/conda-forge/conda-forge-webservices/issues/121 ) on this point.

**Comment by jakirkham** [comment]
Given everything else here is now done and the webpage listing is the only remaining task, which is already documented in issue ( https://github.com/conda-forge/conda-forge-webservices/issues/121 ), closing this issue out.

**Comment by jakirkham** [comment]
The webpage listing has also been broken out now. The bulk of the work was in PR ( https://github.com/conda-forge/conda-forge-webservices/pull/142 ), but some follow-up PRs were necessary.

As such this is now totally complete. 🎉
